test/lb-ng.test.js

```diff
@@ -9,30 +9,29 @@ var parse = require('loopback-sdk-angular/parse-helper');
 
 describe('lb-ng', function() {
   var sampleAppJs = require.resolve('./fixtures/app.js');
+  var sampleAsyncAppJs = require.resolve('./fixtures/async-app/app.js');
   var SANDBOX = path.resolve(__dirname, 'sandbox');
 
   beforeEach(resetSandbox);
 
   it('prints help on no arguments', function() {
-    return runLbNg()
-      .then(function() {
-        throw new Error('lb-ng was supposed to fail with non-zero exit code');
-      })
-      .catch(function(err) {
-        expect(err.code, 'exit code').to.equal(1);
-        expect(err.message).to.contain('Usage');
-      });
+    return runLbNg().then(function() {
+      throw new Error('lb-ng was supposed to fail with non-zero exit code');
+    })
+    .catch(function(err) {
+      expect(err.code, 'exit code').to.equal(1);
+      expect(err.message).to.contain('Usage');
+    });
```

Comment:
Ah, I wouldn't change formatting here. When there is more than one then/spread/whatever block, then the formatting here makes more sense (in my subjective opinion).